packages/fx-core/src/question/create.ts

```diff
@@ -921,7 +922,7 @@ export function capabilityQuestion(): SingleSelectQuestion {
         case ProjectTypeOptions.customCopilot().id:
           return getLocalizedString("core.createProjectQuestion.projectType.customCopilot.title");
         case ProjectTypeOptions.customizeGpt().id:
-          return "Choose the GPT type";
+          return "Choose GPT type";
```

Comment:
Do we also need to replace this GPT with declarative copilot?

Reply:
Thanks, updated.